tools/build_node.js

```diff
@@ -31,6 +31,12 @@ async function buildNodeLanguage(language) {
   await rollupWrite(input, output);
 }
 
+async function buildNodeBundle() {
```

Comment:
Wouldn't `buildNodeLanguage` also have to be updated/suplimented to build all the individual languages in CJS?

Reply:
It depends, we could decide that we supply only a bundle CJS file – or we can go the dual module package way and provide a CJS module for each language, but I thought you said that was not acceptable?